lib/core/sessions.js

```diff
@@ -81,12 +83,7 @@ class ClientSession extends EventEmitter {
         typeof options.causalConsistency !== 'undefined' ? options.causalConsistency : true
     };
 
-    options = options || {};
-    if (typeof options.initialClusterTime !== 'undefined') {
-      this.clusterTime = options.initialClusterTime;
-    } else {
-      this.clusterTime = null;
-    }
+    this.clusterTime = options.initialClusterTime != null ? options.initialClusterTime : null;
```

Comment:
I think we could simplify this even further to `this.clusterTime = options.initialClusterTime`.